src/drivers/riscv_cpu.c

```diff
@@ -105,6 +125,20 @@ void __metal_default_sw_handler (int id, void *priv) {
     }
 }
 
+/* The metal_timer_interrupt_vector_handler() function can be redefined. */
+void __attribute__((weak, interrupt)) metal_timer_interrupt_vector_handler (void) {
```

Comment:
As discussed, just leaving here as a reminder that we should probably create a macro to help define all of these default overridable handlers, since they can be easily de-duplicated.